topics/imaging/tutorials/imaging-introduction/tutorial.md

```diff
@@ -241,9 +241,11 @@ Objects of interest like nuclei can be segmented by using a smoothed image and t
 > 7. **Overlay Images** {% icon tool %} with the following parameters to convert the image to PNG:
 >    - *"How to visualize the overlay?"*: `Segmentation mask over image`
 >    - {% icon param-file %} *"Image"*: `viz_normalized` file
->    - {% icon param-file %} *"Label image"*: `input_segmented` file (output of **Auto Threshold** {% icon tool %})
->    - *"Contour thickness"*: `0.3`
+>    - {% icon param-file %} *"Label image"*: `input_segmented_labeled` file (output of **Binary 2 Label** {% icon tool %})
```

Comment:
While you are in here can you please add tool IDs. This way people can open tools directly from the tutorial.